### ISSUE TITLE
Date and time picker

### ISSUE THREAD

**Issue by dusansimic** [lainsce/countdown#14]
Currently date and time are inputed via regular text boxes (and no notification is displayed if input is incorrect). It would be more convenient to have date and time picker widgets like in GNOME Calendar. Those widgets are [not yet available](https://gitlab.gnome.org/GNOME/libadwaita/-/issues/144) in GTK4 or libadwaita but it could be implemented using the Calendar implementation as inspiration. It would improve the usability of the app greatly.

**Comment by lainsce** [comment]
I agree that the current UI isn't the best it could be, but I don't think copying Calendar in this place would be useful UI, I have an idea in mind that I didn't have time to implement.
